guides/common/attributes.adoc

```diff
@@ -54,6 +57,9 @@ ifeval::["{build}" == "satellite"]
 :foreman-installer: satellite-installer
 :installer-scenario: satellite-installer --scenario satellite
 :installer-scenario-smartproxy: satellite-installer --scenario capsule
+:install: yum install
+:clean: yum clean
+:remove: yum remove
```

Comment:
Isn't this supposed to be `foreman-maintain` now?

Reply:
@lzap Capsule doesn't have satellite-maintain so we need yum some of the time!